diff --git a/cmake/win32_compiler_options.cmake b/cmake/win32_compiler_options.cmake
index ea67deb2f..fe693a125 100644
--- a/cmake/win32_compiler_options.cmake
+++ b/cmake/win32_compiler_options.cmake
@@ -16,6 +16,7 @@ function(pdal_target_compile_settings target)
                 -D_CRT_SECURE_NO_WARNINGS
                 -D_CRT_NONSTDC_NO_WARNING
                 -D_SCL_SECURE_NO_WARNINGS
+                -DNOMINMAX
             )
             target_compile_options(${target} PRIVATE
                 # Yes, we don't understand GCC pragmas
